core/src/main/java/com/aembootcamp/core/models/ArticleModel.java

```diff
@@ -0,0 +1,90 @@
+package com.aembootcamp.core.models;
+
+import lombok.Getter;
+import org.apache.sling.api.SlingHttpServletRequest;
+import org.apache.sling.api.resource.Resource;
+import org.apache.sling.api.resource.ResourceResolver;
+import org.apache.sling.models.annotations.Model;
+import org.apache.sling.models.annotations.DefaultInjectionStrategy;
+import org.apache.sling.models.annotations.injectorspecific.SlingObject;
+import org.apache.sling.models.annotations.injectorspecific.ValueMapValue;
+
+import com.adobe.cq.dam.cfm.ContentFragment;
+import com.day.cq.wcm.api.Page;
+
+import javax.annotation.PostConstruct;
+import javax.inject.Inject;
+
+@Getter
+@Model(adaptables = { Resource.class, SlingHttpServletRequest.class }, resourceType = {
```

Comment:
I don't think that you need the adaptable SlingHttpServletRequest.class. Please remove it and try if the model still works.